### PR TITLE
Rustfmt librustc unicode

### PR DESCRIPTION
**Pull request by little-dude**
I didn't see anything particularly weird here. The last commit has the only changes I'm not 100% sure about. 

I ignored tables.rs since it's genrated, but I updated the python script that generates it.

?r @nrc

### CODE REVIEW

File: src/etc/unicode.py [hunk 1]
```diff
@@ -38,6 +38,7 @@
 // NOTE: The following code was generated by "src/etc/unicode.py", do not edit directly
 
 #![allow(missing_docs, non_upper_case_globals, non_snake_case)]
+// #![rustfmt_skip]
```
Comment (nrc) [inline src/etc/unicode.py]: Should remove rather than comment this.

File: src/librustc_unicode/u_str.rs [hunk 2]
```diff
@@ -72,6 +80,7 @@ impl UnicodeStr for str {
 }
 
 // https://tools.ietf.org/html/rfc3629
+// #[rustfmt_skip]
```
Comment (nrc) [inline src/librustc_unicode/u_str.rs]: and here

**Comment by nrc** [comment]
Looks good, thanks for the PR! r+ with the skip comments removed and the commits squashed.

**Comment by little-dude** [comment]
@nrc thank you :) I fixed according to your comments and rebased.

**Pull request merged**